src/VisualStudio/Core/Def/EditorConfigSettings/SettingsEditorFactory.cs

```diff
@@ -72,6 +74,13 @@ public void Dispose()
                                         out Guid pguidCmdUI,
                                         out int pgrfCDW)
         {
+            if (!_workspace.CurrentSolution.Projects.Any(p => p.Language == LanguageNames.CSharp || p.Language == LanguageNames.VisualBasic))
+            {
+                // If there are no VB or C# projects loaded in the solution (so an editorconfig file in a C++ project) then we want their
+                // editorfactory to present the file instead of use showing ours
+                return VSConstants.VS_E_UNSUPPORTEDFORMAT;
+            }
```

Comment:
Would this prevent the ability of just opening an .editorconfig loose? Or does that already not work for some other reason?

Reply:
>  Or does that already not work for some other reason?

This will also prevent the "loose" problem where you are opening a editorconfig file and there are _no_ projects at all, just files

Reply:
I guess what if I _want_ to open an .editorconfig loose and get the designer? Is that just not supported?